### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:
Some of the links in README is outdated after the documentation overhaul. This PR updates those links to their correct new path. Relate to #482 

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [ ] Appropriate docstring(s) are added/updated
 - [ ] Code is (approximately) PEP8 compliant
 - [ ] Issue(s) raised/addressed?

### CODE REVIEW

**Comment by InnocentBug** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -104,15 +104,15 @@ Example template for disseminating force fields:
 
 
 #### Using Foyer to perform atom typing:
-* [Basic usage examples](docs/usage_examples.rst)
+* [Basic usage examples](docs/source/topic_guides/usage_examples.rst)
 * [Detailed Jupyter notebook tutorials, including integration with mBuild](https://github.com/mosdef-hub/foyer_tutorials)
 * [Jupyter notebook tutorials](https://github.com/mosdef-hub/foyer/tree/master/docs/examples), from [our paper](https://arxiv.org/abs/1812.06779)
```
Comment (jennyfothergill) [inline README.md:109]: I think this link is also broken

**Review by jennyfothergill — COMMENTED** [review]
One change then LGTM

**Review by jennyfothergill — APPROVED** [review]

**Review by justinGilmer — APPROVED** [review]
LGTM!

**Pull request merged**